quantum/action.c

```diff
@@ -689,6 +689,11 @@ void process_action(keyrecord_t *record, action_t action) {
                     }
 #    endif // !defined(NO_ACTION_ONESHOT) && !defined(NO_ACTION_TAPPING)
                     break;
+                case OP_EE_DEFAULT:
```

Comment:
Knowing that we want to remove/refactor/replace this code in the long term, I dont think we should be implementing this within the legacy tmk code.

Instead it would just be a `quantum/process_keycode/process_default_layer.c` and `quantum/process_keycode/process_default_layer.h`.

This would implement a `bool process_default_layer(uint16_t keycode, keyrecord_t *record);` that performs the call to `set_single_persistent_default_layer`.

Reply:
Promoted into its own set of `.c` and `.h` files with a copyright header and hooked into `process_record_quantum` in commit 437cf94d2689dac90db6a62b68299d7b591030fc. I could not update the documentation atomically in this PR; please see my comment below.